### PR TITLE
MS20170: Update Image/Text Entity properties in Create with new properties

### PR DESCRIPTION
**Pull request by thoys**
https://highfidelity.manuscript.com/f/cases/20170/Update-Image-Text-Entity-props-in-Create-with-new-properties

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16272/HighFidelity-Beta-PR14660-26f46e8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16272/HighFidelity-Beta-PR14660-26f46e8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16272/HighFidelity-Beta-Interface-PR14660-26f46e8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16272/HighFidelity-Beta-PR14660-26f46e8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16272/HighFidelity-Beta-Interface-PR14660-26f46e8.exe

**Comment by QliCknox** [comment]
Currently Testing

File: scripts/system/html/js/entityProperties.js [hunk 1]
```diff
@@ -478,6 +480,37 @@ const GROUPS = [
...
+            },
+            {
+                label: "Emissive",
+                type: "bool",
+                propertyID: "emissive",
+            },
+            {
+                label: "Sub Image",
+                type: "rect",
+                min: 0,
```
Comment (huffman) [inline scripts/system/html/js/entityProperties.js]: Do we enforce this minimum in code? I'm wondering if we wrap it

Comment (dback2) [reply scripts/system/html/js/entityProperties.js]: We don't it's signed int

File: scripts/system/assets/data/createAppTooltips.json [hunk 2]
```diff
@@ -20,6 +20,10 @@
     "faceCamera": {
```
Comment (SamGondelman) [inline scripts/system/assets/data/createAppTooltips.json]: You can delete this "faceCamera" tooltip.

File: scripts/system/assets/data/createAppTooltips.json [hunk 3]
```diff
@@ -149,9 +153,25 @@
...
+    "emissive": {
+        "tooltip": "If enabled, the image will display at full brightness."
+    },
+    "subImage": {
+        "tooltip": "The area of the image that is displayed."
+    },
+    "imageBillboardMode": {
+        "tooltip": "If enabled, determines how the entity will face the camera.",
+        "jsPropertyName": "billboardMode"
+    },
```
Comment (SamGondelman) [inline scripts/system/assets/data/createAppTooltips.json]: Do we really need different tooltips for "textBillboardMode" and "imageBillboardMode"?

File: scripts/system/assets/data/createAppTooltips.json (hunk 2 above)
Comment (huffman) [reply scripts/system/assets/data/createAppTooltips.json]: David will address this in his PR

**Pull request merged**